lib/kitchen/directions/bake_further_reading.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Kitchen::Directions::BakeFurtherReading
+  def self.v1(chapter:, metadata_source:, uuid_prefix: '.', klass: 'further-reading')
+    further_reading = Kitchen::Clipboard.new
+
+    chapter.search('section.further-reading').each do |section|
+      section.search('h3').cut
+      section.cut(to: further_reading)
+    end
+
+    Kitchen::Directions::EocCompositePageContainer.v1(
+      container_key: klass,
+      uuid_key: "#{uuid_prefix}#{klass}",
+      metadata_source: metadata_source,
+      content: further_reading.paste,
+      append_to: chapter
+    )
+  end
+end
```

Comment:
This seems like an opportunity to use `MoveCustomSectionToEocContainer`. This direction [can be passed an optional block argument](https://github.com/openstax/kitchen/blob/main/lib/kitchen/directions/move_custom_section_to_eoc_container/v1.rb#L12), which transforms each section before it's moved to the eoc container. This way, basic content transformations con be done from outside the direction. 

Not every eoc section is simple enough to use this -- if the content requires more elaborate transformation (i.e. a wrapper), the best we can do is `EocCompositePageContainer`. But in this case, it looks like this section just needs to remove a redundant title. [Here](https://github.com/openstax/recipes/pull/46/files#diff-b241fef3bc8dbd18e144e07149bfefcbb936d880a72c2ce0fb2487ff14b16c56R32-R41) is an example of a similar thing being done for poli sci. 

The advantage of this approach is that the section can be transformed entirely on the recipe side, and a new direction doesn't have to be made in Kitchen. This is good because it's less work for devs, we don't have to come up with test data, Kitchen doesn't get cluttered with one-off eoc sections, and the recipe won't be held back by needing a Kitchen release for a new eoc section. If a few other books have the same eoc section, though, then we might want to refactor the direction into Kitchen.

tl;dr we probably don't need to add `BakeFurtherReading` to Kitchen just yet if philo is the only book to use it